ndk.rosinstall

```diff
@@ -222,70 +222,70 @@
-# - tar:
-#     local-name: image_transport_plugins/compressed_image_transport
-#     uri: https://github.com/ros-gbp/image_transport_plugins-release/archive/release/kinetic/compressed_image_transport/1.9.5-0.tar.gz
-#     version: image_transport_plugins-release-release-kinetic-compressed_image_transport-1.9.5-0
-# - tar:
-#     local-name: image_transport_plugins/image_transport_plugins
-#     uri: https://github.com/ros-gbp/image_transport_plugins-release/archive/release/kinetic/image_transport_plugins/1.9.5-0.tar.gz
-#     version: image_transport_plugins-release-release-kinetic-image_transport_plugins-1.9.5-0
+- tar:
+    local-name: image_common/camera_calibration_parsers
+    uri: https://github.com/ros-gbp/image_common-release/archive/release/kinetic/camera_calibration_parsers/1.11.13-0.tar.gz
+    version: image_common-release-release-kinetic-camera_calibration_parsers-1.11.13-0
+- tar:
+    local-name: image_common/camera_info_manager
+    uri: https://github.com/ros-gbp/image_common-release/archive/release/kinetic/camera_info_manager/1.11.13-0.tar.gz
+    version: image_common-release-release-kinetic-camera_info_manager-1.11.13-0
+- tar:
+    local-name: image_common/image_common
+    uri: https://github.com/ros-gbp/image_common-release/archive/release/kinetic/image_common/1.11.13-0.tar.gz
+    version: image_common-release-release-kinetic-image_common-1.11.13-0
+- tar:
+    local-name: image_common/image_transport
+    uri: https://github.com/ros-gbp/image_common-release/archive/release/kinetic/image_transport/1.11.13-0.tar.gz
+    version: image_common-release-release-kinetic-image_transport-1.11.13-0
+- tar:
+    local-name: image_common/polled_camera
+    uri: https://github.com/ros-gbp/image_common-release/archive/release/kinetic/polled_camera/1.11.13-0.tar.gz
+    version: image_common-release-release-kinetic-polled_camera-1.11.13-0
+- tar:
+    local-name: image_pipeline/camera_calibration
+    uri: https://github.com/ros-gbp/image_pipeline-release/archive/release/kinetic/camera_calibration/1.12.23-0.tar.gz
+    version: image_pipeline-release-release-kinetic-camera_calibration-1.12.23-0
+- tar:
+    local-name: image_pipeline/depth_image_proc
+    uri: https://github.com/ros-gbp/image_pipeline-release/archive/release/kinetic/depth_image_proc/1.12.23-0.tar.gz
+    version: image_pipeline-release-release-kinetic-depth_image_proc-1.12.23-0
+- tar:
+    local-name: image_pipeline/image_pipeline
+    uri: https://github.com/ros-gbp/image_pipeline-release/archive/release/kinetic/image_pipeline/1.12.23-0.tar.gz
+    version: image_pipeline-release-release-kinetic-image_pipeline-1.12.23-0
+- tar:
+    local-name: image_pipeline/image_proc
+    uri: https://github.com/ros-gbp/image_pipeline-release/archive/release/kinetic/image_proc/1.12.23-0.tar.gz
+    version: image_pipeline-release-release-kinetic-image_proc-1.12.23-0
+- tar:
+    local-name: image_pipeline/image_publisher
```

Comment:
I'm getting an undefined reference when linking here:
```
/opt/ros_android/output/target/lib/libcamera_info_manager.a(camera_info_manager.cpp.o): In function `camera_info_manager::CameraInfoManager::loadCalibrationFile(std::__ndk1::basic_string<char, std::__ndk1::char_traits<char>, std::__ndk1::allocator<char> > const&, std::__ndk1::basic_string<char, std::__ndk1::char_traits<char>, std::__ndk1::allocator<char> > const&)':
/opt/ros_android/output/catkin_ws/src/image_common/camera_info_manager/src/camera_info_manager.cpp:277: undefined reference to `camera_calibration_parsers::readCalibration(std::__ndk1::basic_string<char, std::__ndk1::char_traits<char>, std::__ndk1::allocator<char> > const&, std::__ndk1::basic_string<char, std::__ndk1::char_traits<char>, std::__ndk1::allocator<char> >&, sensor_msgs::CameraInfo_<std::__ndk1::allocator<void> >&)'
/opt/ros_android/output/target/lib/libcamera_info_manager.a(camera_info_manager.cpp.o): In function `camera_info_manager::CameraInfoManager::saveCalibrationFile(sensor_msgs::CameraInfo_<std::__ndk1::allocator<void> > const&, std::__ndk1::basic_string<char, std::__ndk1::char_traits<char>, std::__ndk1::allocator<char> > const&, std::__ndk1::basic_string<char, std::__ndk1::char_traits<char>, std::__ndk1::allocator<char> > const&)':
/opt/ros_android/output/catkin_ws/src/image_common/camera_info_manager/src/camera_info_manager.cpp:546: undefined reference to `camera_calibration_parsers::writeCalibration(std::__ndk1::basic_string<char, std::__ndk1::char_traits<char>, std::__ndk1::allocator<char> > const&, std::__ndk1::basic_string<char, std::__ndk1::char_traits<char>, std::__ndk1::allocator<char> > const&, sensor_msgs::CameraInfo_<std::__ndk1::allocator<void> > const&)'
```
The patch seems to be applied correctly; does the same happen to you?